### PR TITLE
Add drop table, drop table cascade & table dependencies

### PR DESCRIPTION
**Pull request by nirfi**

### CODE REVIEW

File: alooma/alooma.py [hunk 1]
```diff
@@ -934,6 +956,14 @@ def alter_table(self, table_name, columns):
 
         return res
 
+    def get_default_schema(self):
+        schemas = self.get_schemas()
```
Comment (oxmane) [inline alooma/alooma.py]: Why not get this from the output config?
Wouldn't it be a "cheaper" REST call?

File: alooma/alooma.py [hunk 2]
```diff
@@ -909,6 +909,28 @@ def create_table(self, table_name, columns):
 
         return parse_response_to_json(res)
 
+    def drop_table(self, table_name, schema=None, cascade=False):
```
Comment (dhercher) [inline alooma/alooma.py]: I don't think we should use default schema here, seems like a dangerous practice to allow being imprecise
Could we reverse the order of the params so its
 drop_table(self, schema, table_name, cascade=False):

File: alooma/alooma.py [hunk 3]
```diff
@@ -943,6 +973,23 @@ def get_table_names(self, schema=None):
         res = self.__send_request(requests.get, url)
         return parse_response_to_json(res)
 
+    def get_table_dependencies(self, table_name, schema=None):
```
Comment (dhercher) [inline alooma/alooma.py]: Same as above, lets not have a default here and reverse param order?

**Comment by dhercher** [comment]
Looks great!!!!

**Pull request merged**